GoogleSignIn/Tests/Unit/GIDSignInTest.m

```diff
@@ -869,99 +876,89 @@ - (void)testNotHandleWrongPath {
 #pragma mark - Tests - disconnectWithCallback:
 
 // Verifies disconnect calls callback with no errors if access token is present.
-- (void)testDisconnect_accessToken {
+- (void)testDisconnect_accessTokenIsPresent {
   [[[_authorization expect] andReturn:_authState] authState];
   [[[_authState expect] andReturn:_tokenResponse] lastTokenResponse];
   [[[_tokenResponse expect] andReturn:kAccessToken] accessToken];
-  [[[_authorization expect] andReturn:_fetcherService] fetcherService];
```

Comment:
Why remove this and add the below?

Reply:
Seems like you're doing this in each test you update. Let's make sure to update those as well if we make a change during PR review.